osafw-app/App_Code/models/LookupManager.cs

```diff
@@ -111,6 +111,12 @@ public virtual bool updateByTname(string tname, int id, Hashtable item, string m
             throw new ApplicationException("Wrong lookup table name");
         string id_fname = fw.model<LookupManagerTables>().getColumnId(defs);
 
+        //set nullable fields to NULL value if empty
+        Hashtable table_schema = db.tableSchemaFull(tname);
+        foreach (string fld_name in item.Keys.Cast<string>().ToArray())
+            if (Utils.isEmpty(item[fld_name]) && (string)((Hashtable)(table_schema[fld_name]))["is_nullable"] == "1")
```

Comment:
Can you make it:
Utils.f2int( ((Hashtable)(table_schema[fld_name]))["is_nullable"] ) == 1

Reply:
done. As I see from the DB code, table scheme is obtained using the `arrayp()`, so the result values are always strings even with implicit conversion from the `DBList/DBRow` to `ArrayList/Hashtable`.